fcsparser/api.py

```diff
@@ -430,6 +444,12 @@ def reformat_meta(self):
         meta['_channels_'] = df
         meta['_channel_names_'] = self.get_channel_names()
 
+    # Constructs Pandas dataframe
+    def dataframe(self):
```

Comment:
okay if I change method name to_dataframe() else change the method to a property?

(I like method names to be verbs with nouns being reserved for attributes, classes and properties)